code/modules/research/designs/weapon_designs.dm

```diff
@@ -333,3 +333,13 @@
 	reagents_list = list("blood" = 50)
 	build_path = /obj/item/v1_arm_shell
 	category = list("Weapons")
+
+/datum/design/muscle_implant
+	name = "Strong-arm Empowered Musculature Implant"
+	desc = "An implant that enhances your muscles to punch harder and throw people back."
+	id = "muscle_implant"
+	req_tech = list("combat" = 7, "syndicate" = 2, "biotech" = 7)
```

Comment:
I heard in coding chat you might wanted to have raise this to syndi 3, just checking in on that

Reply:
I did say that, and I just forgot to raise it, thanks!